tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,246 @@
+
+Prior to staking tokens, you’ll need the address of the collator you want to delegate to. To retrieve the list of collator candidates available in the network, head to the **Developer** tab, click on **Chain State**, and take the following steps:
+
+ 1. Choose the pallet to interact with. In this case, it is the **parachainStaking** pallet
+ 2. Choose the state to query. In this case, it is the **selectedCandidates** or **candidatePool** state
+ 3. Send the state query by clicking on the **+** button
+
+Each extrinsic provides a different response:
+
+ - **selectedCandidates** — returns the current active set of collators, that is, the top collator candidates by total tokens staked (including delegations). For example, on Moonbase Alpha it is the top {{ networks.moonbase.staking.max_candidates }} candidates
+ - **candidatePool** — returns the current list of all the candidates, including those that are not in the active set
+
+![Staking Account](/images/tutorials/remote-staking-via-xcm/xcm-stake-4.png)
+
+### Get the Candidate Delegation Count {: #get-the-candidate-delegation-count } 
+
+First, you need to get the `candidateInfo`, which will contain the delegator count, as you'll need to submit this parameter in a later transaction. To retrieve the parameter, make sure you're still on the **Chain State** tab of the **Developer** page, and then take the following steps:
+
+ 1. Choose the **parachainStaking** pallet to interact with
+ 2. Choose the **candidateInfo** state to query
+ 3. Make sure the **include option** slider is enabled
+ 4. Enter the collator candidate's address
+ 5. Send the state query by clicking on the **+** button
+ 6. Copy the result as you'll need it when initiating a delegation
+
+![Get candidate delegation count](/images/tutorials/remote-staking-via-xcm/xcm-stake-5.png)
+
+Optionally, you can receive this information via the Polkadot.js API by running the following JavaScript code snippet in [Polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/js){target=_blank}:
+
+```js
+// Simple script to get the collator's number of existing delegations.
+// Remember to replace YOUR-ADDRESS-HERE with your collator address.
+const collatorAddress = 'COLLATOR-ADDRESS-HERE'; 
+const collatorInfo = await api.query.parachainStaking.candidateInfo(collatorAddress);
+console.log(collatorInfo.toHuman()["delegationCount"]);
+```
+
+![Get candidate delegation count](/images/tutorials/remote-staking-via-xcm/xcm-stake-6.png)
+
+### Get your Number of Existing Delegations {: #get-your-number-of-existing-delegations }
+
+If you've never made a delegation from your address you can skip this section. However, if you're unsure how many existing delegations you have, you'll want to run the following JavaScript code snippet to get `delegationCount` from within [Polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/js){target=_blank}:
```

Comment:
I know I made a comment about reusing this section, but maybe only part of it and explicitly state here that they need to get the delegations for the multilocation derivative account